make fails with "dasher_editor_external_atspi.cpp:69:75: error: invalid conversion"

While trying to build on Ubuntu Linux 14.04, `make` failed with this output:
```
libtool: compile:  g++ -DHAVE_CONFIG_H -I. -I../.. -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/x86_64-linux-gnu/dbus-1.0/include -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -pthread -I/usr/include/gtk-3.0 -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/pango-1.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/cairo -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/pixman-1 -I/usr/include/libpng12 -I./../DasherCore -DPROGDATA=\"/usr/local/share/dasher\" -pthread -I/usr/include/gtk-3.0 -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/pango-1.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/cairo -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/pixman-1 -I/usr/include/libpng12 -pthread -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -pthread -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -g -O2 -MT libdashergtk_la-dasher_editor_external_atspi.lo -MD -MP -MF .deps/libdashergtk_la-dasher_editor_external_atspi.Tpo -c dasher_editor_external_atspi.cpp  -fPIC -DPIC -o .libs/libdashergtk_la-dasher_editor_external_atspi.o
dasher_editor_external_atspi.cpp: In function 'void dasher_editor_external_create_buffer(DasherEditor*)':
dasher_editor_external_atspi.cpp:69:75: error: invalid conversion from 'void (*)(AtspiEvent*, void*) {aka void (*)(_AtspiEvent*, void*)}' to 'AtspiEventListenerCB {aka void (*)(const _AtspiEvent*, void*)}' [-fpermissive]
   p->pFocusListener = atspi_event_listener_new(focus_listener, pSelf, NULL);
                                                                           ^
In file included from /usr/include/at-spi-2.0/atspi/atspi.h:38:0,
                 from dasher_editor_external_atspi.cpp:4:
/usr/include/at-spi-2.0/atspi/atspi-event-listener.h:82:1: error:   initializing argument 1 of 'AtspiEventListener* atspi_event_listener_new(AtspiEventListenerCB, gpointer, GDestroyNotify)' [-fpermissive]
 atspi_event_listener_new (AtspiEventListenerCB callback,
 ^
dasher_editor_external_atspi.cpp:70:75: error: invalid conversion from 'void (*)(AtspiEvent*, void*) {aka void (*)(_AtspiEvent*, void*)}' to 'AtspiEventListenerCB {aka void (*)(const _AtspiEvent*, void*)}' [-fpermissive]
   p->pCaretListener = atspi_event_listener_new(caret_listener, pSelf, NULL);
                                                                           ^
In file included from /usr/include/at-spi-2.0/atspi/atspi.h:38:0,
                 from dasher_editor_external_atspi.cpp:4:
/usr/include/at-spi-2.0/atspi/atspi-event-listener.h:82:1: error:   initializing argument 1 of 'AtspiEventListener* atspi_event_listener_new(AtspiEventListenerCB, gpointer, GDestroyNotify)' [-fpermissive]
 atspi_event_listener_new (AtspiEventListenerCB callback,
 ^
make[3]: *** [libdashergtk_la-dasher_editor_external_atspi.lo] Error 1
make[3]: Leaving directory `/home/dbarnett/projects/dasher/Src/Gtk2'
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory `/home/dbarnett/projects/dasher/Src'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/dbarnett/projects/dasher'
make: *** [all] Error 2
```

It looks like the same issue as https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=755343, which was supposedly fixed a year ago. Seeing if I can find a Debian patch that was never accepted upstream or something…

--- a/configure.ac
+++ b/configure.ac
@@ -329,7 +329,7 @@ AC_CHECK_LIB(expat, XML_Parse,,[
 ])
 
 PKG_CHECK_MODULES([ATSPI],
-	[atspi-2],
+	[atspi-2 >= 2.11],
 	[have_libatspi=yes],
 	[have_libatspi=no])
 

